projects/packages/jetpack-mu-wpcom/src/features/blog-privacy/blog-privacy.php

```diff
@@ -18,11 +18,15 @@
 /**
  * Filters the robots.txt contents based on the value of the wpcom_data_sharing_opt_out option.
  *
- * @param string     $output The contents of robots.txt.
- * @param int|string $public The value of the blog_public option.
+ * @param string|null $output The contents of robots.txt.
+ * @param int|string  $public The value of the blog_public option.
  * @return string
  */
-function robots_txt( string $output, $public ): string {
+function robots_txt( ?string $output, $public ): string {
```

Comment:
Given how hooks work, I wonder if we shouldn't just do like
```suggestion
function robots_txt( $output, $public ): string {
```
if we're going to be trying to work around whatever garbage some earlier hook might have returned.